customisation/glossaries.md

```diff
@@ -5,3 +5,45 @@ layout: coming_soon
 title: Glossaries
 description: Terminology lists for training and customising machine translation
 ---
+#Glossaries
+
+Glossaries are lists of terms in one language aligned to their translation in one or more languages. They can be added to machine translation systems.
+Adding a glossary to a commercial machine translation provider can increase the terminology accuracy of the output.
+
+## Application
+
+Glossaries are leveraged at translation time, and no training is required.
+When a glossary term is matched in the source sentence, its translation is enforced in the output.
+
+The insertion of a glossary term in the output happens regardless of the context.
+The terms included in the glossary should therefore be unambiguous and context-independent, as in the following examples:
+- Respiratory syndrome
+- Balance sheet
+- Paraffin
+
+Ambiguous terms can have a negative impact on the output:
+- Spread
+- Cookies
+- Surface treatment
+
+##Term matching and insertion
+
+Several machine translation providers offer the possibility to enhance their output using a glossary.
```

Comment:
Good question. We can't be 100% sure that this is true because many providers do not disclose how the glosary functionality works, so if we try to be more specific, there is a risk that what we say does not apply to (all of?) them. WHat do you think?

Reply:
Then we should remove this whole section or, if we feel it's common enough (I do), just make it

> Glossaries can be applied in two steps:

So "are" -> "can be"